Description: xcb: don't get initial screen rotation
 "xcb_randr_get_screen_info" can be slow and in some configurations can
 cause short mouse cursor freezes (which will happen on Qt application
 startup).
 .
 Initial screen rotation was used only to not handle possible redundant
 screen change event.
Origin: upstream, https://code.qt.io/cgit/qt/qtbase.git/commit/?id=d4e937a6280f34bc
Last-Update: 2018-10-29

--- a/src/plugins/platforms/xcb/qxcbscreen.cpp
+++ b/src/plugins/platforms/xcb/qxcbscreen.cpp
@@ -118,13 +118,6 @@ QXcbVirtualDesktop::QXcbVirtualDesktop(Q
 
         xcb_depth_next(&depth_iterator);
     }
-
-    if (connection->hasXRandr()) {
-        xcb_connection_t *conn = connection->xcb_connection();
-        auto screen_info = Q_XCB_REPLY(xcb_randr_get_screen_info, conn, screen->root);
-        if (screen_info)
-            m_rotation = screen_info->rotation;
-    }
 }
 
 QXcbVirtualDesktop::~QXcbVirtualDesktop()
--- a/src/plugins/platforms/xcb/qxcbscreen.h
+++ b/src/plugins/platforms/xcb/qxcbscreen.h
@@ -220,7 +220,7 @@ private:
     xcb_randr_crtc_t m_crtc;
     xcb_randr_mode_t m_mode = XCB_NONE;
     bool m_primary = false;
-    uint8_t m_rotation = XCB_RANDR_ROTATION_ROTATE_0;
+    uint8_t m_rotation = 0;
 
     QString m_outputName;
     QSizeF m_outputSizeMillimeters;
